build/css/app.css

```diff
@@ -337,10 +340,126 @@ template {
   display: none;
 }
 
+html {
+  box-sizing: border-box;
+  font-size: 62.5%;
+}
+
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+body {
+  font-size: 18px;
+}
+
 body {
   background-color: #55afca;
 }
 
-h1 {
-  color: green;
+.header {
+  height: 100vh;
+}
+@media (min-width: 360px) {
+  .header {
+    background-image: url("/build/img/mobile/image-header.jpg");
+    background-position: top;
+    background-repeat: no-repeat;
+    background-size: contain;
```

Comment:
this line keep it because if you include the line on the background image, as shown above, safari will not recognize the "contain" value, so, this shouldn't be changed